website/docs/d/atracker_routes.html.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_atracker_routes"
+description: |-
+  Get information about atracker_routes
+subcategory: "Activity Tracking API"
+---
+
+# ibm_atracker_routes
+
+Provides a read-only data source for atracker_routes. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
+
+## Example Usage
+
+```hcl
+data "ibm_atracker_routes" "atracker_routes" {
+	name = "my-route"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Optional, string) The name of the route.
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The unique identifier of the atracker_routes.
```

Comment:
Same as above

Reply:
With recent generators tools their are some change in the documentation like the order of arguments in alphabetical order,  change in description of arguments and attributes

Reply:
comment on #1 - this is a uuid not a timestamp.

Reply:
comment on #2 - what actions do I need to take on this? Is it just putting the arguments in alphabetical order?

Reply:
comment #2 - I will get the latest generator code and regenerate.

Reply:
checked in new code with generator 3.37.0